frog/imports/client/Wiki/WikiContentComp.js

```diff
@@ -130,20 +151,30 @@ class WikiContentComp extends React.Component<> {
           Finish
         </Button>
       );
-    })();
+    };
 
     const titleDiv = this.state.editingTitle ? (
       <div style={titleDivStyle}>
-        <div>
-          <input
+        <FormControl margin="normal">
```

Comment:
Seems like you are also making UI changes, while trying to fix a bug? Is this integral to fixing the bug? Otherwise would have been better to have two separate PRs, easier to test/review.

Reply:
I needed to add helper text to the input to display the errors. I felt it was better to use the Material UI's TextField and FormHelperText than the HTML input to make it more uniform with all of the other UI, so I decided to include that in. I could make a separate PR for this but then for this PR (if I wanted to use the standard HTML input) I would have to use the standard HTML label and include some CSS to display the error properly. Regardless things would have to change on the UI side of things. Let me know what you think.

Reply:
That's totally fine if it's integral to fixing the bug. Thanks